applications/FluidDynamicsApplication/tests/CylinderTest/cylinder_fluid_parameters.json

```diff
@@ -144,5 +144,25 @@
             "remove_output_file"    : true,
 	        "decimal_places"      : 7
         }
```

Comment:
This test existed before. Should it also work when there is not HDF5 support?

Reply:
Yes. I was hoping I could avoid duplicating the json file for each test, but I don't see an easy way of achieving that.

Reply:
you could read it and "hack" the parameter object
I do sth similar with the restart tests in StructuralMechanics, which are based on existing tests
see [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/tests/restart_tests.py#L43)

Reply:
@philbucher thanks for the suggestion, I've gone for something similar to your example.